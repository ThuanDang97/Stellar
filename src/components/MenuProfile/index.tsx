import { FC, memo, useCallback } from 'react'
import { useTheme } from 'styled-components'
import { useRouter } from 'next/router'
import { useSWRConfig } from 'swr'
import Link from 'next/link'

// Props types
import { CacheSWRProps } from '@self-types/swr'
import { MenuProps } from '@self-types/components/Menu.props'

// Components
import {
  ListItemStyled,
  ListStyled,
} from '@components/styled-components/ListStyled'
import { WrapperAvatarStyled } from '@layouts/Header/HeaderStyled'
import LinkComponent from '@components/Link'
import Avatar from '@components/Avatar'
import Button from '@components/Button'
import {
  BoxShadowAnimationStyled,
  TextStyled,
  ViewStyled,
} from '@components/styled-components'

// Constants
import {
  DEFAULT_HEADER_URL,
  LOCAL_STORAGE_KEY,
  MENU_STATUS,
  PRIMARY_HEADER_URL,
} from '@constants/index'

// Utils
import { pxToRem } from '@utils/theme'
import { clearLocalStorageItem } from '@utils/localStorage'
import { useAuthContext } from '@hooks/useAuthContext'

const MenuProfile: FC<MenuProps> = ({
  imageUrl,
  fullName,
  menuStatus,
  setMenuStatus,
}) => {
  const theme = useTheme()
  const router = useRouter()
  const { setUserId } = useAuthContext()
  const { cache } = useSWRConfig()
  const cacheFetchingData = cache as CacheSWRProps
  const handleLogout = useCallback(() => {
    setMenuStatus(MENU_STATUS.NONE)
    clearLocalStorageItem(LOCAL_STORAGE_KEY.USER_ID)
    setUserId('')
    cacheFetchingData.clear()
    router.push(DEFAULT_HEADER_URL.LOGIN.URL)
  }, [cacheFetchingData, router, setMenuStatus, setUserId])

  const closeMenu = useCallback(() => {
    setMenuStatus(MENU_STATUS.NONE)
  }, [setMenuStatus])

  const handleClickDashboardMenu = () => {
    setMenuStatus(MENU_STATUS.NONE)
    if (router.pathname === PRIMARY_HEADER_URL.DASHBOARD.URL) router.reload()
  }

  return (
    <BoxShadowAnimationStyled
      data-testid="menuHeader"
      textAlign="center"
      flexDirection="column"
      alignItems="initial"
      flexWrap="wrap"
      position="absolute"
      width={theme.metrics.width.xl}
      bgColor={theme.colors.white}
      zIndex={theme.metrics.zIndex.top}
      top={50}
      right={1}
      borderRadius={theme.metrics.borderRadius.default}
      duration={0.3}
      fadeInOut={menuStatus}
    >
      <ListStyled width={theme.metrics.width.full}>
        <ListItemStyled bgColorHover={theme.colors.white} pTop={0} pBottom={0}>
          <WrapperAvatarStyled
            columnGap={theme.metrics.dimensions.base}
            display="flex"
            borderBottom={`${pxToRem(
              theme.metrics.borderWidth.default,
            )} solid ${theme.colors.silverLight}`}
            pBottom={theme.metrics.dimensions.md}
            pTop={theme.metrics.dimensions.md}
            mLeft={theme.metrics.dimensions.md}
            mRight={theme.metrics.dimensions.md}
          >
            <Avatar
              width={theme.metrics.dimensions.xxl}
              height={theme.metrics.dimensions.xxl}
              fullName={fullName}
              imageUrl={imageUrl}
            />
            <ViewStyled
              display="flex"
              flexDirection="column"
              alignItems="flex-start"
            >
              <TextStyled
                fontWeight={theme.typography.fontWeight.bold}
                textOverflow="ellipsis"
                whiteSpace="nowrap"
                overflow="hidden"
                maxWidth={130}
              >
                {fullName}
              </TextStyled>
              <LinkComponent
                href={PRIMARY_HEADER_URL.USER_PROFILE.URL}
                bgColorHover={theme.colors.ironLight}
                text="See Your Profile"
                display="inline"
                fontSize={theme.typography.fontSize.xss}
                color={theme.colors.bigStone}
                textAlign="left"
                pTop={theme.metrics.dimensions.xs}
                pBottom={theme.metrics.dimensions.xs}
                fontFamily="AdobeCleanLight"
                letterSpacing={theme.typography.letterSpacing.xxl}
                onClick={closeMenu}
              />
            </ViewStyled>
          </WrapperAvatarStyled>
        </ListItemStyled>
        <Link href={PRIMARY_HEADER_URL.REFER_A_FRIEND.URL} passHref>
          <ListItemStyled
            textAlign="left"
            pTop={0}
            pBottom={0}
            bgColorHover={theme.colors.ironLight}
            onClick={closeMenu}
          >
            <ViewStyled
              borderBottom={`${pxToRem(
                theme.metrics.borderWidth.default,
              )} solid ${theme.colors.silverLight}`}
              pTop={theme.metrics.dimensions.md}
              pBottom={theme.metrics.dimensions.md}
              mLeft={theme.metrics.dimensions.md}
              mRight={theme.metrics.dimensions.md}
            >
              <LinkComponent
                onClick={closeMenu}
                href={PRIMARY_HEADER_URL.REFER_A_FRIEND.URL}
                linkTypes="forgotPasswordLink"
                text="Refer A Friend"
                display="inline"
                fontSize={theme.typography.fontSize.sm}
                fontWeight={theme.typography.fontWeight.bold}
              />
            </ViewStyled>
          </ListItemStyled>
        </Link>
        <Link
          href={{
            pathname: DEFAULT_HEADER_URL.CONNECT_BANK.URL,
            query: { returnUrl: router.route },
          }}
          passHref
        >
          <ListItemStyled
            textAlign="left"
            pTop={0}
            pBottom={0}
            bgColorHover={theme.colors.ironLight}
            onClick={closeMenu}
          >
            <ViewStyled
              borderBottom={`${pxToRem(
                theme.metrics.borderWidth.default,
              )} solid ${theme.colors.silverLight}`}
              pTop={theme.metrics.dimensions.md}
              pBottom={theme.metrics.dimensions.md}
              mLeft={theme.metrics.dimensions.md}
              mRight={theme.metrics.dimensions.md}
            >
              <LinkComponent
                onClick={closeMenu}
                href={{
                  pathname: DEFAULT_HEADER_URL.CONNECT_BANK.URL,
                  query: { returnUrl: router.route },
                }}
                linkTypes="forgotPasswordLink"
                text="Connect Bank"
                display="inline"
                fontSize={theme.typography.fontSize.sm}
                fontWeight={theme.typography.fontWeight.bold}
              />
            </ViewStyled>
          </ListItemStyled>
        </Link>
        <ListItemStyled
          textAlign="left"
          bgColorHover={theme.colors.white}
          pTop={0}
          pBottom={0}
          mTop={theme.metrics.dimensions.sm}
          onClick={closeMenu}
        >
          <ListStyled
            display="block"
            width={theme.metrics.width.full}
            onClick={closeMenu}
          >
            <Link href={PRIMARY_HEADER_URL.DASHBOARD.URL} passHref>
              <ListItemStyled
                pTop={0}
                pBottom={0}
                bgColorHover={theme.colors.ironLight}
                onClick={closeMenu}
              >
                <LinkComponent
                  mLeft={theme.metrics.dimensions.md}
                  href={PRIMARY_HEADER_URL.DASHBOARD.URL}
                  linkTypes="forgotPasswordLink"
                  text="Dashboard"
                  display="inline"
                  fontSize={theme.typography.fontSize.sm}
                  fontWeight={theme.typography.fontWeight.bold}
                  onClick={handleClickDashboardMenu}
                />
              </ListItemStyled>
            </Link>
            <Link href={PRIMARY_HEADER_URL.HELP.URL}>
              <ListItemStyled
                pTop={0}
                pBottom={0}
                bgColorHover={theme.colors.ironLight}
                onClick={closeMenu}
              >
                <LinkComponent
                  onClick={closeMenu}
                  href={PRIMARY_HEADER_URL.HELP.URL}
                  linkTypes="forgotPasswordLink"
                  text="Help & Support"
                  display="inline"
                  fontSize={theme.typography.fontSize.sm}
                  fontWeight={theme.typography.fontWeight.bold}
                  mLeft={theme.metrics.dimensions.md}
                />
              </ListItemStyled>
            </Link>
            <ListItemStyled
              pTop={0}
              pBottom={0}
              bgColorHover={theme.colors.ironLight}
              onClick={closeMenu}
            >
              <Button
                fontSize={theme.typography.fontSize.sm}
                fontWeight={theme.typography.fontWeight.bold}
                color={theme.colors.shark}
                variant="dark"
                size="default"
                title="Log Out"
                padding={1}
                textAlign="start"
                data-testid="buttonLogout"
                mLeft={theme.metrics.dimensions.md}
                onClick={handleLogout}
              />
            </ListItemStyled>
          </ListStyled>
        </ListItemStyled>
        <ListItemStyled bgColorHover={theme.colors.white}>
          <TextStyled textAlign="start" pLeft={theme.metrics.dimensions.base}>
            <LinkComponent
              bgColorHover={theme.colors.ironLight}
              padding={theme.metrics.dimensions.xs}
              href={PRIMARY_HEADER_URL.PRIVACY_POLICY.URL}
              onClick={closeMenu}
              text="Privacy Policy"
              display="inline"
              mLeft={theme.metrics.dimensions.tiny}
              fontSize={theme.typography.fontSize.xss}
              textAlign="left"
              fontFamily="AdobeCleanLight"
              letterSpacing={theme.typography.letterSpacing.xs}
              color={theme.colors.spanishGray}
            />
            <TextStyled color={theme.colors.spanishGray}>{` - `}</TextStyled>
            <LinkComponent
              onClick={closeMenu}
              href={PRIMARY_HEADER_URL.TERMS_OF_USE.URL}
              padding={theme.metrics.dimensions.xs}
              bgColorHover={theme.colors.ironLight}
              text="Terms of Use"
              display="inline"
              mLeft={theme.metrics.dimensions.tiny}
              fontSize={theme.typography.fontSize.xss}
              textAlign="left"
              fontFamily="AdobeCleanLight"
              letterSpacing={theme.typography.letterSpacing.xs}
              color={theme.colors.spanishGray}
            />
          </TextStyled>
        </ListItemStyled>
      </ListStyled>
    </BoxShadowAnimationStyled>
  )
}

export default memo(MenuProfile)
