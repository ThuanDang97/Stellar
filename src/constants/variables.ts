import { TextScoreProps } from '@self-types/components/FicoScoreProps.props'
import Colors from '@themes/Colors'

export const SIZE_BUTTON = {
  DEFAULT: 'default',
  MEDIUM: 'medium',
  LARGE: 'large',
  NO_PADDING: 'noPadding',
}

export const VARIANT_BUTTON = {
  PRIMARY: 'primary',
  NO_BORDER: 'noBorder',
  DARK: 'dark',
  LIGHT: 'light',
  DANGER: 'danger',
  DISABLED: 'disabled',
  ICON: 'icon',
  WARNING: 'warning',
  HALF_CIRCLE: 'half-circle',
}

export const LIST_BILLER = [
  {
    id: 'b1',
    src: '/images/netflix.png',
    title: 'Netflix',
  },
  {
    id: 'b2',
    src: '/images/t-mobile.png',
    title: 'T-Mobile',
  },
  {
    id: 'b3',
    src: '/images/hulu.png',
    title: 'Hulu',
  },
  {
    id: 'b4',
    src: '/images/x-finity.png',
    title: 'Xfinity',
  },
  {
    id: 'b5',
    src: '/images/at-wireless.png',
    title: 'AT&T Wireless',
  },
  {
    id: 'b6',
    src: '/images/spotify.png',
    title: 'Spotify',
  },
]

export const LIST_GOAL = [
  {
    id: 'g1',
    url: '/icons/clock',
    title: 'Improve My \n Credit Score',
    activeText: 'Improve My \n Credit Score',
  },
  {
    id: 'g2',
    url: '/icons/build-wealth',
    title: 'Build \n Wealth',
    activeText: 'Pay My Bills \n In One Place',
  },
  {
    id: 'g3',
    url: '/icons/flag',
    title: 'Buy A Car',
    activeText: 'Buy A Car',
  },
  {
    id: 'g4',
    url: '/icons/job',
    title: 'Get A Job',
    activeText: 'Get A Job',
  },
  {
    id: 'g5',
    url: '/icons/house',
    title: 'Buy \n A Home',
    activeText: 'Buy \n A Home',
  },
  {
    id: 'g6',
    url: '/icons/rent',
    title: 'Rent \n A Home',
    activeText: 'Rent \n A Home',
  },
  {
    id: 'g7',
    url: '/icons/credit-card',
    title: 'Get A \n Credit Card',
    activeText: 'Get A \n Credit Card',
  },
  {
    id: 'g8',
    url: '/icons/insurance-rate',
    title: 'Get Better \n Insurance Rate',
    activeText: 'Get Better \n Insurance Rate',
  },
  {
    id: 'g9',
    url: '/icons/interest-rate',
    title: 'Get Better \n Interest Rates',
    activeText: 'Get Better \n Interest Rates',
  },
  {
    id: 'g10',
    url: '/icons/bill',
    title: 'Pay May Bills \n On Time',
    activeText: 'Improve \n My Finances',
  },
  {
    id: 'g11',
    url: '/icons/piggy-bank',
    title: 'Get A Loan',
    activeText: 'Get A Loan',
  },
  {
    id: 'g12',
    url: '/icons/business',
    title: 'Start \n A Business',
    activeText: 'Start \n A Business',
  },
]

export const LABEL_BOX = [
  { id: 'labelBox1', title: 'Podcast' },
  { id: 'labelBox2', title: 'Referral' },
  { id: 'labelBox3', title: 'Clubhouse' },
  { id: 'labelBox4', title: 'Pinterest' },
  { id: 'labelBox5', title: 'YouTube' },
  { id: 'labelBox6', title: 'Facebook' },
  { id: 'labelBox7', title: 'Online Ad' },
  { id: 'labelBox8', title: 'Wed Search' },
  { id: 'labelBox9', title: 'Instagram' },
  { id: 'labelBox10', title: 'TikTok' },
  { id: 'labelBox11', title: 'Other' },
]

export const STEPPER = [
  { id: 'signup', step: 'Sign Up' },
  { id: 'goal', step: 'Set Goals' },
  { id: 'connectbank', step: 'Connect Bank' },
]

export const FLEX_LIST_FEATURE = [
  { feature: 'Everything in Launch', isActive: true },
  { feature: 'Flexible Bill pay', isActive: true },
  { feature: 'All 3 Credit Reports', isActive: false },
  { feature: 'Credit Lock', isActive: false },
  { feature: 'Credit Alerts', isActive: false },
  { feature: 'Credit Score Factors', isActive: false },
  { feature: 'Debt Analysis', isActive: false },
]

export const LIST_LAUNCH_FEATURE = [
  { feature: 'Credit Builder Bill Pay', isActive: true },
  { feature: 'Credit Score Monitoring', isActive: false },
  { feature: 'Credit Score Simulator', isActive: false },
  { feature: 'Shopping Deals & Offers', isActive: false },
]

export const FORMAT_DATE = 'MM/DD/YYYY'

export const LOCAL_STORAGE_KEY = {
  USER_ID: 'userId',
  IS_TOKEN: 'is-token',
}

export const OPACITY_BLUR = 0.2
export const OPACITY_BOLD = 1

export const POOR_SCORE: TextScoreProps = {
  color: Colors.pomegranate,
  text: 'Poor',
}

export const FAIR_SCORE: TextScoreProps = {
  color: Colors.pizzaz,
  text: 'Fair',
}

export const GOOD_SCORE: TextScoreProps = {
  color: Colors.supernova,
  text: 'Good',
}

export const EXCELLENT_SCORE: TextScoreProps = {
  color: Colors.caribbeanGreen,
  text: 'Excellent',
}

export const MIN_AGE_OF_USER = 18

export const MAX_AGE_OF_USER = 150

export const FIRST_LETTER_NAME_OF_MONTHS = [
  'J',
  'F',
  'M',
  'A',
  'M',
  'J',
  'J',
  'A',
  'S',
  'O',
  'N',
  'D',
]

export const SSN_LENGTH = 9

export const COUNTRY_CODE = '+1'

export const LIST_PAYMENTS_BY_MONTH = [
  {
    value: '1',
    name: 'Pay Monthly',
  },
  {
    value: '3',
    name: 'Pay 3 months',
  },
  {
    value: '6',
    name: 'Pay 6 months',
  },
  {
    value: '12',
    name: 'Pay Annually',
  },
]

export const MONTH_ABBREVIATE = [
  'Jan',
  'Feb',
  'Mar',
  'Apr',
  'May',
  'Jun',
  'Jul',
  'Aug',
  'Sep',
  'Oct',
  'Nov',
  'Dec',
]

export const FREE_CREDIT_SCORE = 708
export const CREDIT_SCORE_LOCKED = 300

export const MENU_STATUS = {
  FADE_IN: 'fadeIn',
  FADE_OUT: 'fadeOut',
  NONE: 'none',
}

export const MAX_LENGTH_INPUT = 12
