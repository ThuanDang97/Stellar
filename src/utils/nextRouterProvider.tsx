import { NextRouter } from 'next/router'
import { RouterContext } from 'next/dist/shared/lib/router-context'

const createMockRouter = (router?: Partial<NextRouter>): NextRouter => ({
  basePath: '',
  pathname: '/',
  route: '/',
  query: {},
  asPath: '/',
  back: jest.fn(),
  beforePopState: jest.fn(),
  prefetch: jest.fn(),
  push: jest.fn(),
  reload: jest.fn(),
  replace: jest.fn(),
  events: {
    on: jest.fn(),
    off: jest.fn(),
    emit: jest.fn(),
  },
  isFallback: false,
  isLocaleDomain: false,
  isReady: true,
  defaultLocale: 'en',
  domainLocales: [],
  isPreview: false,
  ...router,
})

export const NextRouterProvider = ({
  children,
  router = { query: {} },
}: {
  children: React.ReactNode
  router: Partial<NextRouter>
}) => (
  <RouterContext.Provider value={createMockRouter(router)}>
    {children}
  </RouterContext.Provider>
)
