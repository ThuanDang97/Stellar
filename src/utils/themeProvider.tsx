import { ThemeProvider } from 'styled-components'
import { render } from '@testing-library/react'
import { theme } from '@themes/index'

export const renderWithTheme = (component: React.ReactNode) => {
  return render(<ThemeProvider theme={theme}>{component}</ThemeProvider>)
}
