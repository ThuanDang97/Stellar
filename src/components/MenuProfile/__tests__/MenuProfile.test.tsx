import React from 'react'

// Components
import MenuProfile from '@components/MenuProfile'

// Utils
import { renderWithTheme } from '@utils/themeProvider'
import { NextRouterProvider } from '@utils/nextRouterProvider'
import { MENU_STATUS } from '@constants/variables'

describe('MenuProfile Component', () => {
  const component = (
    <NextRouterProvider router={{ route: '/abc' }}>
      <MenuProfile
        fullName=""
        menuStatus={MENU_STATUS.FADE_OUT}
        setMenuStatus={jest.fn()}
      />
    </NextRouterProvider>
  )

  test('Component MenuProfile matchers DOM Snapshot', () => {
    const { container } = renderWithTheme(component)

    expect(container).toMatchSnapshot()
  })
})
