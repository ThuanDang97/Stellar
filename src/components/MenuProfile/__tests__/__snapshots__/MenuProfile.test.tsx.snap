// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuProfile Component Component MenuProfile matchers DOM Snapshot 1`] = `
<div>
  <div
    class="ViewStyled-sc-rqe5sa-0 ViewStyled__BoxShadowStyled-sc-rqe5sa-2 ViewStyled__BoxShadowAnimationStyled-sc-rqe5sa-3 exDRui csavJt bSwOhO"
    data-testid="menuHeader"
    width="300"
  >
    <ul
      class="ListStyled-sc-t03hvr-0 hFGRnl"
      width="100%"
    >
      <li
        class="ListStyled__ListItemStyled-sc-t03hvr-1 ggeHjz"
      >
        <div
          class="ViewStyled-sc-rqe5sa-0 HeaderStyled__WrapperAvatarStyled-sc-ht65yv-2 fkNXGy faxsfr"
          display="flex"
        >
          <div
            class="ViewStyled-sc-rqe5sa-0 dEUOMm"
            cursor="pointer"
            height="60"
            width="60"
          >
            <span
              class="AvatarStyled-sc-1ppokaq-1 bOiqfJ"
              data-testid="avatarDefault"
            />
          </div>
          <div
            class="ViewStyled-sc-rqe5sa-0 iMoZMg"
            display="flex"
          >
            <span
              class="TextStyled-sc-9xia41-0 evMDQc"
              font-weight="700"
              overflow="hidden"
            />
            <a
              class="LinkStyled__ButtonLinkStyled-sc-626068-1 fbACSi"
              color="#101F30"
              data-testid="link"
              display="inline"
              font-family="AdobeCleanLight"
              font-size="13"
              href="/profile"
              letter-spacing="0.9"
            >
              See Your Profile
            </a>
          </div>
        </div>
      </li>
      <li
        class="ListStyled__ListItemStyled-sc-t03hvr-1 dBsbXq"
        href="/refer-a-friend"
      >
        <div
          class="ViewStyled-sc-rqe5sa-0 kQGDDR"
        >
          <a
            class="LinkStyled__ButtonLinkStyled-sc-626068-1 iIGnIj"
            data-testid="link"
            display="inline"
            font-size="16"
            font-weight="700"
            href="/refer-a-friend"
          >
            Refer A Friend
          </a>
        </div>
      </li>
      <li
        class="ListStyled__ListItemStyled-sc-t03hvr-1 dBsbXq"
        href="/connect-bank?returnUrl=%2Fabc"
      >
        <div
          class="ViewStyled-sc-rqe5sa-0 kQGDDR"
        >
          <a
            class="LinkStyled__ButtonLinkStyled-sc-626068-1 iIGnIj"
            data-testid="link"
            display="inline"
            font-size="16"
            font-weight="700"
            href="/connect-bank?returnUrl=%2Fabc"
          >
            Connect Bank
          </a>
        </div>
      </li>
      <li
        class="ListStyled__ListItemStyled-sc-t03hvr-1 eXKxno"
      >
        <ul
          class="ListStyled-sc-t03hvr-0 jAJdQE"
          display="block"
          width="100%"
        >
          <li
            class="ListStyled__ListItemStyled-sc-t03hvr-1 koFPQW"
            href="/dashboard"
          >
            <a
              class="LinkStyled__ButtonLinkStyled-sc-626068-1 fyaZWH"
              data-testid="link"
              display="inline"
              font-size="16"
              font-weight="700"
              href="/dashboard"
            >
              Dashboard
            </a>
          </li>
          <li
            class="ListStyled__ListItemStyled-sc-t03hvr-1 koFPQW"
          >
            <a
              class="LinkStyled__ButtonLinkStyled-sc-626068-1 fyaZWH"
              data-testid="link"
              display="inline"
              font-size="16"
              font-weight="700"
              href="/help"
            >
              Help & Support
            </a>
          </li>
          <li
            class="ListStyled__ListItemStyled-sc-t03hvr-1 koFPQW"
          >
            <button
              class="ButtonStyled-sc-1ys9v52-0 gwnvOP"
              color="#242A30"
              data-testid="buttonLogout"
              font-size="16"
              font-weight="700"
            >
              Log Out
            </button>
          </li>
        </ul>
      </li>
      <li
        class="ListStyled__ListItemStyled-sc-t03hvr-1 jfbuwf"
      >
        <span
          class="TextStyled-sc-9xia41-0 jjPAoQ"
        >
          <a
            class="LinkStyled__ButtonLinkStyled-sc-626068-1 bvoeHr"
            color="#919099"
            data-testid="link"
            display="inline"
            font-family="AdobeCleanLight"
            font-size="13"
            href="/privacy-policy"
            letter-spacing="0.4"
          >
            Privacy Policy
          </a>
          <span
            class="TextStyled-sc-9xia41-0 hrVQLR"
            color="#919099"
          >
             - 
          </span>
          <a
            class="LinkStyled__ButtonLinkStyled-sc-626068-1 bvoeHr"
            color="#919099"
            data-testid="link"
            display="inline"
            font-family="AdobeCleanLight"
            font-size="13"
            href="/terms-of-use"
            letter-spacing="0.4"
          >
            Terms of Use
          </a>
        </span>
      </li>
    </ul>
  </div>
</div>
`;
